src/worldmap.ts

```diff
@@ -181,14 +201,14 @@ export default class WorldMap {
   }
 
   calcCircleSize(dataPointValue) {
-    const circleMinSize = parseInt(this.ctrl.panel.circleMinSize, 10) || 2;
+    let circleMinSize = parseInt(this.ctrl.panel.circleMinSize, 10) || 2;
```

Comment:
Why did you change it to `let`?

If there is a valid reasoning for that then shouldn't we have to change `circleMaxSize` as well?

Reply:
This was something I did on the first iteration of this development, that I chose to undo because it was too ugly. Forgot to roll THAT back ... done! :D